src/applications/vaos/tests/project-cheetah/components/ClinicChoicePage.unit.spec.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { expect } from 'chai';
+import { waitFor } from '@testing-library/dom';
+import { cleanup } from '@testing-library/react';
+import {
+  createTestStore,
+  renderWithStoreAndRouter,
+  setVaccineFacility,
+} from '../../mocks/setup';
+import userEvent from '@testing-library/user-event';
+
+import ClinicChoicePage from '../../../project-cheetah/components/ClinicChoicePage';
+import { mockEligibilityFetches } from '../../mocks/helpers';
+import { getClinicMock } from '../../mocks/v0';
+import { mockFetch, resetFetch } from 'platform/testing/unit/helpers';
+import { TYPE_OF_CARE_ID } from '../../../project-cheetah/utils';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingVSPAppointmentNew: false,
+    vaOnlineSchedulingDirect: true,
+  },
+  user: {
+    profile: {
+      facilities: [{ facilityId: '983', isCerner: false }],
+    },
+  },
+};
+
+describe('VAOS vaccine flow <ClinicChoicePage>', () => {
+  beforeEach(() => mockFetch());
+  afterEach(() => resetFetch());
+  it('should display multiple clinics and require one to be chosen', async () => {
+    const clinics = [
+      {
+        id: '308',
+        attributes: {
+          ...getClinicMock(),
+          siteCode: '983',
+          clinicId: '308',
+          institutionCode: '983',
+          clinicFriendlyLocationName: 'Green team clinic',
+        },
+      },
+      {
+        id: '309',
+        attributes: {
+          ...getClinicMock(),
+          siteCode: '983',
+          clinicId: '309',
+          institutionCode: '983',
+          clinicFriendlyLocationName: 'Red team clinic',
+        },
+      },
+    ];
+    mockEligibilityFetches({
+      siteId: '983',
+      facilityId: '983',
+      typeOfCareId: TYPE_OF_CARE_ID,
+      clinics,
+    });
+
+    const store = createTestStore(initialState);
+
+    await setVaccineFacility(store, '983', {
+      name: 'Cheyenne VA Medical Center',
+      address: {
+        physical: {
+          zip: '82001-5356',
+          city: 'Cheyenne',
+          state: 'WY',
+          address1: '2360 East Pershing Boulevard',
+        },
+      },
+      phone: {
+        main: '307-778-7550',
+      },
+    });
+
+    const screen = renderWithStoreAndRouter(<ClinicChoicePage />, {
+      store,
+    });
+
+    await screen.findByText(/Choose a clinic located at/i);
+
+    expect(screen.baseElement).to.contain.text('Cheyenne VA Medical Center');
+    expect(screen.baseElement).to.contain.text('Cheyenne, WY 82001-5356');
+    expect(screen.baseElement).to.contain.text('307-778-7550');
+
+    expect(screen.getAllByRole('radio').length).to.equal(2);
+    expect(screen.getByLabelText('Green team clinic')).to.have.tagName('input');
+    expect(screen.getByLabelText('Red team clinic')).to.have.tagName('input');
```

Comment:
Is tagName more of an implementation detail?

Reply:
Yeah, should really search for these by role.